src/README.md.jinja

```diff
@@ -12,7 +12,12 @@
     {{- version | replace(".","-") -}}
 {%- endmacro -%}
 [![Runbot Status](https://runbot.odoo-community.org/runbot/badge/flat/{{ repo_id(repo_slug) }}/{{ odoo_version }}.svg)](https://runbot.odoo-community.org/runbot/repo/github-com-oca-{{ repo_slug }}-{{ repo_id(repo_slug) }})
+{%- if ci == "Travis" %}
 [![Build Status](https://travis-ci.com/OCA/{{ repo_slug }}.svg?branch={{ odoo_version }})](https://travis-ci.com/OCA/{{ repo_slug }})
+{%- else %}
+[![Pre-commit Status](https://github.com/OCA/{{ repo_slug }}/actions/workflows/pre-commit.yml/badge.svg?branch={{ odoo_version }})](https://github.com/OCA/{{ repo_slug }}/actions/workflows/pre-commit.yml?branch={{ odoo_version }})
+[![Build Status](https://github.com/OCA/{{ repo_slug }}/actions/workflows/test.yml/badge.svg?branch={{ odoo_version }})](https://github.com/OCA/{{ repo_slug }}/actions/workflows/test.yml?branch={{ odoo_version }})
```

Comment:
Testing it, the URL is not totally correct. It should be like

https://github.com/OCA/l10n-spain/actions/workflows/test.yml?query=branch%3A15.0

Reply:
Thanks! I followed Github's documentation on this, but indeed this seems like a better option :+1: